addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/PiiScanRule.java

```diff
@@ -252,6 +256,23 @@ public List<Alert> getExampleAlerts() {
                         .build());
     }
 
+    private boolean isMessageOfType(HttpMessage msg, String fileType, String contentType) {
```

Comment:
It occurred to me this usage could be made less generic. I’ll tweak it in the morning. Please go ahead and review the rest.